moveit_planners/chomp/chomp_interface/src/chomp_interface.cpp

```diff
@@ -69,6 +69,7 @@ void CHOMPInterface::loadParams()
   nh_.param("collision_threshold", params_.collision_threshold_, 0.07);
   // nh_.param("random_jump_amount", params_.random_jump_amount_, 1.0);
   nh_.param("use_stochastic_descent", params_.use_stochastic_descent_, true);
-  nh_.param("interpolation_method", params_.trajectory_initialization_method_, std::string("quintic-spline"));
+  nh_.param("trajectory_initialization_method", params_.trajectory_initialization_method_,
+            std::string("quintic-spline"));
```

Comment:
note this breaks API.... @bmagyar thoughts?

Reply:
I don't think CHOMP has such a large userbase to hold off on this one. It makes much clearer what that param is for. 

:+1: